### ISSUE TITLE
Automatically installing dependencies

### ISSUE THREAD

**Issue by ghost** [perrette/papers#8]
```
pip install git+https://github.com/perrette/papers.git#egg=papers
```
doesn't automatically get the dependencies. Looks like [`setup.py`](https://github.com/perrette/papers/blob/master/setup.py) is using `distutils.core`, which I don't know very well. Perhaps there's a way to do this in distutils as well, but in setuptools, `setuptools.setup(install_requires=)` can automatically get dependencies when installing.

Docs here:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#install-requires

**Comment by perrette** [comment]
Thanks for your comment, i will take a look when I am seriously back on this project, but not before another couple of months. For now, the project is more experimental. I intend to make a pip version so that this sort of issue should not be relevant anymore.

**Comment by perrette** [comment]
I think this should have been fixed in v2. Closing for now.

**Comment by perrette** [comment]
Nope. Not fixed. Soon.

**Comment by perrette** [comment]
Fixed in f84f346a11add7d1651b4e3313aaedc9120d1e6e
